src/data/constants.js

```diff
@@ -0,0 +1,3 @@
+export const moviePageMainTitle = "Popular Movies";
+
+export const noItemsMessage = "No items were found that match your query.";
```

Comment:
A more descriptive name indicating an error would be helpful, `NO_ITEMS_ERROR` maybe?

Reply:
about this constant NO_ITEMS_MESSAGE. It's not really an error. It's a message that appears when no movies appear based on the user chosen filters. Should I still change it name to NO_ITEMS_ERROR?

Reply:
In that case, it would be fine to keep it, we'll just need to convert it to uppercase